hazelcast/src/main/java/com/hazelcast/util/CollectionUtil.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (c) 2008-2015, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.util;
+
+import java.util.Collection;
+
+/**
+ * Various collection utility methods, mainly targeted to use internally.
+ */
+public final class CollectionUtil {
+
+    private CollectionUtil() {
+    }
+
+    /**
+     * Returns {@code true} if the given collection is null or empty, otherwise returns {@code false}.
+     *
+     * @param collection the given collection.
+     * @return {@code true} if collection is empty.
+     */
+    public static boolean isEmpty(Collection collection) {
+        return collection == null || collection.isEmpty();
+    }
+
+    /**
+     * Returns {@code true} if the given collection is not null and not empty, otherwise returns {@code false}.
+     *
+     * @param collection the given collection.
+     * @return {@code true} if collection is not empty.
+     */
+    public static boolean isNotEmpty(Collection collection) {
+        return !isEmpty(collection);
```

Comment:
We probably don't need `isNotEmpty` since `!isEmpty` is shorter. Negated version can be created for every boolean method:)